backend/app/assets/stylesheets/spree/backend/shared/_forms.scss

```diff
@@ -42,4 +42,13 @@ input.form-control {
 .fullwidth-input {
   display: block;
   width: 100%;
+}
+
+.help-block {
+  color: #A6A6A6;
```

Comment:
Color literals like `#A6A6A6` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#A6A6A6` should be written as `#a6a6a6`